src/coprocessor/dag/expr/builtin_cast.rs

```diff
@@ -55,7 +55,10 @@ impl FnCall {
         if val.is_none() {
             return Ok(None);
         }
-        let val = val.unwrap().round(0, RoundMode::HalfEven).unwrap();
+        let val = val.unwrap()
+            .into_owned()
```

Comment:
Because you can't modify `Cow` directly. We need clone the content before do that.